recipes/protobuf/all/conanfile.py

```diff
@@ -52,6 +53,22 @@ def _is_clang_x86(self):
     def _can_disable_rtti(self):
         return Version(self.version) >= "3.15.4"
 
+    @property
+    def _min_cppstd(self):
+        return 11 if Version(self.version) < "4.22.0" else 14
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "14": {
+                "gcc": "8",
```

Comment:
I might be wrong, but abseil and protobuf both support gcc 7.3 as minimal

Reply:
fixed.

Reply:
Thanks!